packages/ccc/package.json

```
@@ -0,0 +1,24 @@
+{
+  "name": "@moeru-ai/ccc",
```

```suggestion
  "name": "@proj-airi/moeru-ai-ccc",
```

Avoid naming conflict